docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -315,3 +315,20 @@ The following migration example allows users with the `Editor` role to access th
                   - identifier: activity_log_owner
                     values: []
 ```
+
+## Update Elasticsearch schema
+
+Elasticsearch index must be erased, the schema updated, and the index rebuild.
+
+To delete the index, for example, you can use an HTTP request like in the following `curl` command:
+
+```bash
+curl --request DELETE 'http://elasticsearch:9200/_all'
```

Comment:
```suggestion
curl --request DELETE 'https://elasticsearch:9200/_all'
```
I know this is just an example, but maybe we should use https in examples as well?